tox.ini

```diff
@@ -0,0 +1,45 @@
+[tox]
+envlist = py{27,37}, docs
+skip_missing_interpreters = true
+
+
+[testenv]
+passenv = SSL_CERT_FILE CI TRAVIS* APPVEYOR* CODECOV*
+setenv = COVERAGE_FILE={toxworkdir}/.coverage.{envname}
+usedevelop = True
+deps = -rrequirements.txt
+       -rtest/requirements.txt
+commands = pytest {posargs:-vv --cov-report term-missing --cov=sureal/}
```

Comment:
This here tells tox what to do to run the tests:
- usedevelop: installs our own project here as `pip editable` (that's customary)
- deps: dependencies to install (here we're getting them from our 2 req files)
- commands: here I went with pytest because of the nicer output, but this could be easily changed to run `unittest` instead